tests/integration/cocoon/test_cocoon_holdings.py

```diff
@@ -44,6 +44,35 @@ def setUpClass(cls) -> None:
                 ["Prime Broker"],
                 "operations001",
                 None,
+                False,
+                lusid.models.Version,
+            ],
+            [
+                "Standard successful load with adjustment only",
+                "prime_broker_test",
+                "data/holdings-example-unique-date.csv",
+                {
+                    "code": "FundCode",
+                    "effective_at": "Effective Date",
+                    "tax_lots.units": "Quantity",
+                },
+                {
+                    "tax_lots.cost.amount": None,
+                    "tax_lots.cost.currency": "Local Currency Code",
+                    "tax_lots.portfolio_cost": None,
+                    "tax_lots.price": None,
+                    "tax_lots.purchase_date": None,
+                    "tax_lots.settlement_date": None,
+                },
+                {
+                    "Isin": "ISIN Security Identifier",
+                    "Sedol": "SEDOL Security Identifier",
+                    "Currency": "is_cash_with_currency",
+                },
+                ["Prime Broker"],
+                "operations001",
+                None,
+                True,
```

Comment:
Can it handle attempting to adjust holdings for holdings that don't exist yet?

Reply:
Yep. LUSID can do this.